test/modules/gltf/gltf/gltf-draco.spec.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable max-len */
+import test from 'tape-catch';
+
+import {loadBinaryFile} from '@loaders.gl/core';
+import {GLTFBuilder, GLTFParser} from '@loaders.gl/gltf';
+import {DracoEncoder, DracoDecoder} from '@loaders.gl/draco';
+import path from 'path';
+
+const POSITIONS =
+  loadBinaryFile(path.resolve(__dirname, '../../../data/raw-attribute-buffers/lidar-positions.bin')) ||
+  require('test-data/raw-attribute-buffers/lidar-positions.bin');
+
+const COLORS =
+  loadBinaryFile(path.resolve(__dirname, '../../../data/raw-attribute-buffers/lidar-positions.bin')) ||
+  require('test-data/raw-attribute-buffers/lidar-colors.bin');
+
+test('GLTFBuilder#addCompressedPointCloud', t => {
+  const attributes = {
+    POSITIONS: new Float32Array(POSITIONS),
+    COLORS: new Uint8ClampedArray(COLORS)
+  };
+  t.comment(`Encoding ${attributes.POSITIONS.length} positions, ${attributes.COLORS.length} colors...`);
+
+  // Encode mesh
+  const options = {
```

Comment:
Just inline these in the constructor call on line 30 instead of assigning to variable?